systemtests/tests/config-dump/etc/bareos/bareos-dir-full.conf.in

```diff
@@ -255,7 +255,7 @@ JobDefs {
 
 JobDefs {
   Name = "JobDefWithRunScripts"
-  # Description = 
```

Comment:
Last time I checked, the trailing spaces have been required for the systemtest to work.

Reply:
Do you remember in what way the test relied on the trailing spaces ? The testrunner seems to run successfully even with them removed.

Reply:
If the config-dump test now succeeds, my hint has just been outdated.